lib/cocoapods/generator/copy_xcframework_script.rb

```diff
@@ -161,7 +161,7 @@ def script
   local basepath="$1"
   local name="$2"
   local package_type="$3"
-  local paths=("${@:3}")
```

Comment:
Can you point to or add a comment that gives an example of how these four arguments are parsed?

Reply:
yes basically:
```
install_xcframework "${PODS_ROOT}/AppCenter/AppCenter-SDK-Apple/AppCenterCrashes.xcframework" "AppCenter/Crashes" "framework" "ios-arm64_arm64e_armv7_armv7s" "ios-arm64_x86_64-maccatalyst" "ios-arm64_i386_x86_64-simulator"
```

We used to pass _all_ arguments including "PODS_ROOT...." and "framework".

Those are being processed by the `select_slice` method.

While in most cases this is fine and causes no issues (except extra cycles to process bad paths) this did cause an actual issue https://github.com/CocoaPods/CocoaPods/issues/10430

In the issue `PODS_ROOT` had x86_64 within its _folder_ name throwing off our `select_slice` script and ultimately failing the build.

This change ensures we grab remainder of args after "framework". In a previous PR of mine I did `@:3` which was fixing #10430 but we were still passing "framework" so I was off by one.